sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java

```diff
@@ -139,7 +143,7 @@ public void onStart() {
     }
 
     @Override
-    public void onCancel(@NonNull DialogInterface dialogInterface) {
+    public void onCancel(@androidx.annotation.NonNull DialogInterface dialogInterface) {
```

Comment:
Why the type change?

Reply:
Because this is the original annotation on the `onCancel`, So Android Studio proposed to change it from `lombok`'s to `@androidx.annotation.NonNull`, Probably we need to change it on `onCreateView` too